graphql/batch_executor.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/samsarahq/thunder/reactive"
 )
 
+type executeFunction func(ctx context.Context) error
```

Comment:
Need to make this a public function I think.  We currently have a public field that has a private type.